### PR TITLE
chore(ui-ux): improve ux on step 2 (dfc -> evm)

### LINKED ISSUE #562: Error code 3: when user send more than specified amount
```
Ver: 0.14.0
Network: TestNet
Flow: DFC → ETH 
Description: Error code 3: when user send more than specified amount
```

**Actual:**
– at **step 3 Validate**, if user encounters `Error code 3: BalanceNotMatched` 
– The balance in the wallet's address does not equal to the amount filled in the form
– user has no other option but to select x and restart the tx 

#### What you expected to happen:
TBC

**To reproduce:**
Select Source Network: Ethereum
Input `0.02` dETH > select `Review transaction`
at **step 1 Initiate**, paste `Ethereum address` in text field > Continue
at **step 2 Transact**, use MLW to send `0.3` dETH > when tx is sent > go back to **Quantum**, select `Verify transfer`
at **step 3 Validate**, `error:` Error code 3: BalanceNotMatched > `note:` user get back 0.1 dETH as specified amount is 0.02 dETH

#### Anything else we need to know?:
230224 Quantum - over sends amount

### PR DESCRIPTION
**Pull request by pierregee**
#### What this PR does / why we need it:
- as titled

#### Which issue(s) does this PR fixes?:

Fixes #562

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] No console errors on web
- [ ] Tested on Light mode and Dark mode\*
- [x] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*
- [ ] Added translations\*

### CODE REVIEW

File: apps/web/src/components/commons/AlertInfoMessage.tsx [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function AlertInfoMessage({
   containerStyle,
   textStyle,
 }: {
-  message: string;
+  message: string | JSX.Element;
```
Comment (fullstackninja864) [inline apps/web/src/components/commons/AlertInfoMessage.tsx:9]: can we avoid passing JSX.Element as props? we can use children property here

File: apps/web/src/styles/globals.scss [hunk 2]
```diff
@@ -185,7 +185,6 @@ body {
   }
   .dark-bg-card-section {
     background: var(--ui-dark-card-section);
-    opacity: var(--tw-bg-opacity);
```
Comment (fullstackninja864) [inline apps/web/src/styles/globals.scss:188]: why its removed?

**Review by fullstackninja864 — CHANGES_REQUESTED** [review]

File: apps/web/src/styles/globals.scss (hunk 2 above)
Comment (pierregee) [reply apps/web/src/styles/globals.scss:188]: Yes, first image is with ` opacity: var(--tw-bg-opacity);` as you can see there's a diff on the colors (looks dim), if I don't remove it, the amount won't be prominent enough for the user to focus their eyes on
image
image

File: apps/web/src/components/commons/AlertInfoMessage.tsx (hunk 1 above)
Comment (pierregee) [reply apps/web/src/components/commons/AlertInfoMessage.tsx]: Updated!

**Pull request merged**